exporters/otlp/common/src/main/java/io/opentelemetry/exporter/internal/otlp/logs/InstrumentationScopeLogsMarshaler.java

```diff
@@ -9,42 +9,42 @@
 import io.opentelemetry.exporter.internal.marshal.MarshalerUtil;
 import io.opentelemetry.exporter.internal.marshal.MarshalerWithSize;
 import io.opentelemetry.exporter.internal.marshal.Serializer;
-import io.opentelemetry.exporter.internal.otlp.InstrumentationLibraryMarshaler;
+import io.opentelemetry.exporter.internal.otlp.InstrumentationScopeMarshaller;
 import io.opentelemetry.proto.logs.v1.internal.InstrumentationLibraryLogs;
 import java.io.IOException;
 import java.util.List;
 
-final class InstrumentationLibraryLogsMarshaler extends MarshalerWithSize {
-  private final InstrumentationLibraryMarshaler instrumentationLibrary;
+final class InstrumentationScopeLogsMarshaler extends MarshalerWithSize {
+  private final InstrumentationScopeMarshaller instrumentationScope;
   private final List<Marshaler> logMarshalers;
   private final byte[] schemaUrlUtf8;
 
-  InstrumentationLibraryLogsMarshaler(
-      InstrumentationLibraryMarshaler instrumentationLibrary,
+  InstrumentationScopeLogsMarshaler(
+      InstrumentationScopeMarshaller instrumentationScope,
       byte[] schemaUrlUtf8,
       List<Marshaler> logMarshalers) {
-    super(calculateSize(instrumentationLibrary, schemaUrlUtf8, logMarshalers));
-    this.instrumentationLibrary = instrumentationLibrary;
+    super(calculateSize(instrumentationScope, schemaUrlUtf8, logMarshalers));
+    this.instrumentationScope = instrumentationScope;
     this.schemaUrlUtf8 = schemaUrlUtf8;
     this.logMarshalers = logMarshalers;
   }
 
   @Override
   public void writeTo(Serializer output) throws IOException {
     output.serializeMessage(
-        InstrumentationLibraryLogs.INSTRUMENTATION_LIBRARY, instrumentationLibrary);
+        InstrumentationLibraryLogs.INSTRUMENTATION_LIBRARY, instrumentationScope);
     output.serializeRepeatedMessage(InstrumentationLibraryLogs.LOG_RECORDS, logMarshalers);
     output.serializeString(InstrumentationLibraryLogs.SCHEMA_URL, schemaUrlUtf8);
   }
 
   private static int calculateSize(
-      InstrumentationLibraryMarshaler instrumentationLibrary,
+      InstrumentationScopeMarshaller instrumentationScope,
       byte[] schemaUrlUtf8,
       List<Marshaler> logMarshalers) {
     int size = 0;
     size +=
         MarshalerUtil.sizeMessage(
-            InstrumentationLibraryLogs.INSTRUMENTATION_LIBRARY, instrumentationLibrary);
+            InstrumentationLibraryLogs.INSTRUMENTATION_LIBRARY, instrumentationScope);
```

Comment:
Just merged the proto update, so think you need to merge main and update the keys now when marshaling

Reply:
Actually, it looks like the proto PR to change the field names hasn't merged yet: https://github.com/open-telemetry/opentelemetry-proto/pull/362